lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components",
-  "version": "3.6.1",
+  "version": "4.0.0",
```

Comment:
Could you clarify where you were unable to keep things backwards compatible and why?

Reply:
> Could you clarify where you were unable to keep things backwards compatible and why?

- removed `<footer>` and `.nx-footer` from the horizontal cards and replaced it with `.nx-card__actions`.
- removed the alternate vertical layout.
- all vertical layouts now have all content within `.nx-card__content`.

It's possible that I could have made the changes that I felt were necessary without causing breaking changes, but my concern was that we'd end up with convoluted and difficult to maintain (and modify) code. Better in my opinion to try and take what we learned from Expedition teams' work and do it right.